board_console_implementation.cpp

```diff
@@ -0,0 +1,90 @@
+#include "board_console_implementation.h"
+
+const int undefinedPerspective = -1;
+
+namespace printBoardAuxiliars {
+	void showPositionStatus(Board const &board, unsigned int x, unsigned int y, int perspective = undefinedPerspective);
+
+	void printRow(Board const &board, unsigned int y);
+
+	void printColumnInRow(Board const &board, unsigned int x, unsigned int y, int perspective);
+}
+
+void printBoardAuxiliars::showPositionStatus(Board const &board, unsigned int x, unsigned int y, int perspective) {
+	// print the status of the position {x, y}
+	//(" " = no existe, "O" = posicion en juego/estado desconocido, "M" = mina del jugador en la posicion)
+	bool isPositionEnabled = true;
+
+	for (int i = 0; i < board.disabledPositions.size(); i++) {
```

Comment:
#should use a range-based for loop, `for (Position const& disabledPos : board.disabledPositions)`